src/node_file.cc

```diff
@@ -384,7 +384,9 @@ int FileHandle::ReadStart() {
       // Lifetime is ensured via AsyncWrap::resource_.
       Local<Object> resource = Object::New(env()->isolate());
       resource->Set(
-          env()->context(), env()->handle_string(), read_wrap->object());
+          env()->context(),
+          env()->handle_string(),
+          read_wrap->object()).Check();
```

Comment:
There doesn’t seem to be a way to make a Github suggestion for this, but please don’t use `.Check()` it that check can fail in new code.

I’d wrap the call here in `USE()`, that should be okay here.